### ISSUE TITLE
Tooltip wraps the contents with 'w-fit' which messes up my layout

### ISSUE THREAD

**Issue by sgehrman** [themesberg/flowbite-react#215]
<Tooltip> does this => <div class="w-fit" data-testid="tooltip-target"> 

The tooltip shouldn't do anything to change the layout.

**Comment by sgehrman** [comment]
I see it's fixed in main.  I'm trying to build a local copy...

I can't even npm i on your code without --force.

update your packages.

 @floating-ui/react-dom                  ^0.6.3  →    ^0.7.1     
 @floating-ui/react-dom-interactions     ^0.4.0  →    ^0.6.3     
 react-icons                             ^4.3.1  →    ^4.4.0     
 @babel/core                            ^7.17.9  →   ^7.18.2     
 @commitlint/cli                        ^16.2.3  →   ^17.0.2     
 @commitlint/config-conventional        ^16.2.1  →   ^17.0.2     
 @storybook/addon-a11y                  ^6.4.22  →    ^6.5.8     
 @storybook/addon-actions               ^6.4.22  →    ^6.5.8     
 @storybook/addon-essentials            ^6.4.22  →    ^6.5.8     
 @storybook/addon-interactions          ^6.4.22  →    ^6.5.8     
 @storybook/addon-links                 ^6.4.22  →    ^6.5.8     
 @storybook/builder-webpack5            ^6.4.22  →    ^6.5.8     
 @storybook/manager-webpack5            ^6.4.22  →    ^6.5.8     
 @storybook/react                       ^6.4.22  →    ^6.5.8     
 @storybook/testing-library              ^0.0.9  →   ^0.0.11     
 @testing-library/react                 ^13.1.1  →   ^13.3.0     
 @testing-library/user-event            ^13.5.0  →   ^14.2.0     
 @types/node                          ^16.11.27  →  ^17.0.41     
 @types/react                           ^18.0.5  →  ^18.0.12     
 @types/react-dom                       ^18.0.1  →   ^18.0.5     
 @types/react-syntax-highlighter        ^13.5.2  →   ^15.5.1     
 @typescript-eslint/eslint-plugin       ^5.27.0  →   ^5.27.1     
 @typescript-eslint/parser              ^5.27.0  →   ^5.27.1     
 @vitest/ui                             ^0.14.0  →   ^0.14.1     
 autoprefixer                           ^10.4.4  →   ^10.4.7     
 axe-core                                ^4.4.1  →    ^4.4.2     
 babel-loader                            ^8.2.4  →    ^8.2.5     
 cypress                                 ^9.6.0  →   ^10.0.3     
 flowbite                                ^1.4.2  →    ^1.4.7     
 husky                                   ^7.0.4  →    ^8.0.1     
 lint-staged                            ^12.3.8  →   ^13.0.1     
 postcss                                ^8.4.12  →   ^8.4.14     
 prettier-plugin-tailwindcss             ^0.1.8  →   ^0.1.11     
 react                                  ^18.0.0  →   ^18.1.0     
 react-dom                              ^18.0.0  →   ^18.1.0     
 react-element-to-jsx-string             14.3.4  →    15.0.0     
 react-scripts                           ^5.0.0  →    ^5.0.1     
 standard-version                        ^9.3.2  →    ^9.5.0     
 storybook-dark-mode                     ^1.0.9  →    ^1.1.0     
 typescript                              ^4.6.3  →    ^4.7.3     
 vite                                    ^2.9.9  →   ^2.9.10     
 vitest                                 ^0.13.1  →   ^0.14.1     
 webpack                                ^5.72.0  →   ^5.73.0

**Comment by tulup-conner** [comment]
Well, you should be using yarn with this project not npm, which will resolve your issue. But yeah, they need to updated regardless.

**Comment by tulup-conner** [comment]
Also, I don't see anything in your original comment, it just says, like this, and points to nothing

**Comment by rluders** [comment]
Hey @sgehrman.

I'll recommend you to check [this discussion](https://github.com/themesberg/flowbite-react/discussions/213).

> TLDR; The `0.0.25` doesn't support a lot of new things that are in master. Next release will be the `1.0.0-alpha`, 'till then, no new release, but you can install it from the github if you really need it.
